docs/ballerina-by-example/examples/fork-join/fork-join.bal

```diff
@@ -27,7 +27,7 @@ function main (string[] args) {
         // Get values received from worker 'w1'.
         any[] resW1 =? <any[]>results["w1"];
         int iW1 =? <int>resW1[0];
-        string sW1 =? <string>resW1[1];
+        string sW1 = <string>resW1[1];
```

Comment:
Casting to string has become a safe operation by design or by accident?

Reply:
Was under the impression it was by design. @lankavitharana can you please confirm.

Reply:
I check against the latest build and behavior is still the same

Reply:
@afkham It is by design. This was discussed in https://groups.google.com/d/topic/ballerina-dev/sQkpK7iW32g/discussion